msdk-featdet/msdk-featdet-ADAP-3D/src/main/java/io/github/msdk/featdet/ADAP3D/common/algorithms/Peak3DTest.java

```diff
@@ -0,0 +1,285 @@
+/*
+ * (C) Copyright 2015-2017 by MSDK Development Team
+ *
+ * This software is dual-licensed under either
+ *
+ * (a) the terms of the GNU Lesser General Public License version 2.1 as published by the Free
+ * Software Foundation
+ *
+ * or (per the licensee's choosing)
+ *
+ * (b) the terms of the Eclipse Public License v1.0 as published by the Eclipse Foundation.
+ */
+package io.github.msdk.featdet.ADAP3D.common.algorithms;
+
+import org.apache.commons.collections4.map.MultiKeyMap;
+
+import java.lang.Math;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * <p>
+ * Peak3DTest is used for testing a peak by comparing adjacent m/z-slices in profile data.
+ * </p>
+ */
+public class Peak3DTest {
+	
+	/**
+	 * <p>
+	 * Result class is used for returning lower and upper mz bound,boolean good peak value and list of similarity value.
```

Comment:
Because in sparse matrix representation we're using int value of mz by multiplying 10000. So we're getting rounded mz values of upper and lower mz bounds.